packages/mobile/src/components/lineup-tile/TrackTile.tsx

```diff
@@ -78,6 +80,11 @@ const TrackTileComponent = ({
   const dispatchWeb = useDispatchWeb()
   const navigation = useNavigation()
   const currentUserId = useSelectorWeb(getUserId)
+  const playingUid = useSelector(getPlayingUid)
```

Comment:
This selector will cause every track tile to rerender when the playingUid changes. Using the selector that was in `LineupTile` prevents this! 

``` 
const isPlayingUid = useSelector(
    (state: AppState) => getPlayingUid(state) === uid
  )
```

Reply:
Right, good point. Will update 👌